src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private Random random = new Random();
+    private Color[] colors = Color.values();
+
+    public Color getRandomColor() {
+        int randomIndex = random.nextInt(colors.length);
+        return colors[randomIndex];
```

Comment:
Do not create redundant variables when you can inline code like this:
```suggestion
        return colors[random.nextInt(colors.length)];
```